src/lib/Libifl/int_status.c

```diff
@@ -572,6 +573,15 @@ PBSD_status_aggregate(int c, int cmd, char *id, void *attrib, char *extend, int
 					cur = last;
 				}
 			}
+		} else {
+			if (!single_itr && (pbs_errno == PBSE_UNKQUE || pbs_errno == PBSE_UNKRESVID)) {
```

Comment:
combine the 'else' and the 'if' to a 'else if'

Reply:
Done